subjects/cybersecurity/osint-master/README.md

```diff
@@ -0,0 +1,183 @@
+
+By completing this project, You will:
+
+- Develop an understanding of OSINT techniques and their applications.
+- Gain practical experience in programming, API integration, and data handling.
+- Learn to identify and mitigate security risks, including subdomain takeovers.
+- Understand the ethical and legal implications of cybersecurity practices.
+
+### Resources
+
+Some useful resources:
+
+- [Open-source intelligence](https://en.wikipedia.org/wiki/Open-source_intelligence)
+- [Doxing](https://en.wikipedia.org/wiki/Doxing)
+- [Kali Tools - Recon](https://en.kali.tools/all/?category=recon)
+- [OSINT Tools on GitHub](https://github.com/topics/osint-tools)
+
+Before asking for help, ask yourself if you have really thought about all the possibilities.
+
+### Role play
+
+To enhance the learning experience and assess your knowledge, a role play question session will be included as part of this project.
+This section will involve answering a series of questions in a simulated real-world scenario where you assume the role of a Cyber Security Expert explaining how to protect information from OSINT techniques to a team or stakeholder.
+
+The goal of the role play question session is to:
+
+- Assess your understanding of OSINT risks and mitigation strategies.
+- Test your ability to communicate effectively and explain security measures related to this project.
+- Challenge you to think critically about the importance of information security and consider alternative approaches.
+- Explain what subdomain takeovers are.
+
+Prepare for a role play question session in the audit.
+
+### Project Requirements
+
+#### Input Handling:
+
+The tool should accept the following inputs: `Full Name`, `IP Address`, `Username`, and `Domain`.
+
+#### Information Retrieval:
+
+- Full Name:
+  Parse the input to extract "First Name" and "Last Name".
+  Look up associated information such as phone numbers, addresses, and social media profiles using directory APIs or web scraping.
+
+- IP Address:
+  Retrieve geolocation data, ISP details, and check for any historical data associated with the IP (e.g., from abuse databases).
+
+- Username:
+  Check for the presence of the username on at least five known social networks and public repositories.
+  Retrieve public profile information, such as profile bio, activity status, and follower count.
+
+- Domain and Subdomain Enumeration:
+  Enumerate subdomains and gather information including IP addresses, SSL certificate details, and potential vulnerabilities.
+  Identify potential subdomain takeover risks by analyzing DNS records and associated resources.
+
+> You are responsible for choosing the way you want to find the data and for using the API's. be aware of the API's `Terms of Use` and `Cost` before use!
```

Comment:
`Be aware ...` - `be` should be capital letter

Reply:
fixed